option java_outer_classname = "PiHeaderProto";
option optimize_for = LITE_RUNTIME;

message PiHeader {
	required uint32 messageLength = 1;
	required uint32 parserID = 2;
	optional uint32 messageID = 3;
	optional uint32 flags = 4;
	optional bool	successResponse = 5;
}